web/src/components/HelpText.js

```diff
@@ -8,10 +8,12 @@ const HelpText = ({ text: textResource, reminder: reminderResource }) => {
   const reminder = t(reminderResource, { defaultValue: false });
 
   return (
-    <div className="mb2" style={{ whiteSpace: 'pre-line' }}>
-      {text && <div>{text}</div>}
-      {reminder && <div className="red">{reminder}</div>}
-    </div>
+    (text || reminder) && (
+      <div className="mb2" style={{ whiteSpace: 'pre-line' }}>
+        {text && <div>{text}</div>}
+        {reminder && <div className="red">{reminder}</div>}
+      </div>
+    )
```

Comment:
Makes the `<HelpText>` component empty if there's no help text or reminder text.  This way we can use it liberally and not worry about the text exists in the resource file, basically putting in placeholders that magically get filled in if the content need arises.